purchases/src/test/java/com/revenuecat/purchases/common/AppConfigTest.kt

```diff
@@ -31,6 +32,7 @@ class AppConfigTest {
             store = Store.PLAY_STORE
         )
         assertThat(appConfig.languageTag).isEqualTo(expected)
+        unmockkStatic("com.revenuecat.purchases.common.UtilsKt")
```

Comment:
Should we move this to a `@Before` and `@After` block? Seems like it's repeated in a lot of tests

Reply:
Yes, I agree